docs/pages/careers/react-tech-lead-core.md

```diff
@@ -115,12 +115,12 @@ We're looking for someone with strong front-end skills. More important than spec
   You have worked on enterprise components in the past, maybe for internal use in your company.
   You have built complex and advanced features while making sure the accessibility was fully functional.
 
-### What it would be nice if you had, but isn't required
+### Nice to have (but not required)
 
 - **You've maintained an active repository before**.
   Maybe you've helped maintain a popular open-source repository, or perhaps you've worked on internal repositories that saw contributions from multiple teams.
   Previous experience with highly active repository workflows is a definite plus for this role.
-- **You have contributed code to MUI before**. A history of contributing to MUI would be a definite plus.
+- **You have contributed code**. A history of contributing to MUI's open-source projects is a plus.
```

Comment:
Fixed a number of confusion sentences where MUI is used as meaning an open-source project. For now, let's keep it to being an organization only, this will make it simpler to mass rename things if we ever change this in the future.